examples/worker/WORKSPACE

```diff
@@ -29,6 +29,11 @@ load("@build_bazel_rules_nodejs//:index.bzl", "yarn_install")
 
 yarn_install(
     name = "npm",
+    # Set this to true in a real world example (the default value is true), to have hermetic
+    # builds and use the `--frozen-lockfile`. We have to disable this here as the @bazel
+    # scoped packages are getting replaced in the package.json with a file url and the version
+    # won't match with the one in the yarn.lock file then.
+    frozen_lockfile = False,
```

Comment:
adding these to the actual code isn't needed, it's a bunch of extra explanation for users, and when they `cd` into this folder to run the example, `frozen_lockfile` will work fine.
We have a plan to fix the integration test runner to use `.tar` file urls which I think will work with frozen lockfile. Therefore the bug is local to the integration test runner, so the workaround should live there too.

Here's the spot where we already modify this file as we copy it into the test fixture
https://github.com/bazelbuild/rules_nodejs/blob/3.x/internal/bazel_integration_test/test_runner.js#L222

Can you just make the change to `yarn_install` there? can be syntactically trivial edit like
`yarn_install(\n` -> `yarn_install(\n    frozen_lockfile=False,\n`

Then we shouldn't need to change any examples

Reply:
@alexeagle Thx for the hint – yea I prefer this a lot 👍  Change already applied ;)